### DISCUSSION TITLE
Disconnessione continua Home Assistant

### DISCUSSION THREAD

**Discussion by Valerio0188** [indomus/forum#2664] [HUB: Home Assistant]
Buongiorno a tutti,

mi rivolgo a voi nella speranza di trovare una soluzione a un problema che sto riscontrando con la mia installazione di Home Assistant.

Ho installato Home Assistant OS su una macchina virtuale che gira su un PC con Windows 11 AMD Ryzen 5560U, 16 GB RAM e SSD da 512 GB. La mia connessione internet è una fibra FTTH 1 Gbit. Il PC è connesso alla mia rete domestica tramite Wi-Fi 5 GHz.

Il problema che sto riscontrando è il seguente: Home Assistant si disconnette periodicamente quando non visualizzo la pagina di Home Assistant. Quando torno alla pagina, vedo il messaggio “Connessione persa. Riconnessione in corso” e dopo pochi secondi si riconnette. Questo accade indipendentemente dal fatto che io stia utilizzando attivamente il PC o meno.

Ho riavviato il router, ma il problema persiste. Tutti gli altri dispositivi sulla rete Wi-Fi funzionano correttamente, quindi sembra che il problema sia limitato a Home Assistant.

Ho anche provato a installare Home Assistant su un altro PC nella stessa rete, ma collegato via Ethernet invece che Wi-Fi. Con questo PC, Home Assistant non perde la connessione, il che mi fa pensare che il problema potrebbe essere legato in qualche modo alla connessione Wi-Fi (?).

Ho provato a disinstallare Home Assistant, cancellare la macchina virtuale e ripartire da zero (seguendo pedissequamente le istruzioni fornite sul sito ufficiale di HA), ma il problema si ripresenta comunque, quindi il problema potrebbe non essere legato ad una specifica installazione di Home Assistant, ma piuttosto a qualcosa nella mia configurazione o nell’interazione tra Home Assistant e la mia rete Wi-Fi.

Spero che queste informazioni vi diano un’idea del problema e che qualcuno possa aiutarmi a risolvere il problema perché di fatto in questo modo HA è inutilizzabile.

Grazie in anticipo per il vostro aiuto.

**Comment by KC-inDomus** [comment]
domanda cretina: non è che ti va in sleep il mini pc

**Reply by Valerio0188** [reply]
Ciao, no, assolutamente. Succede anche mentre sto utilizzando il mini PC.
Ad esempio, stamattina lo stavo usando per navigare su internet e scrivere un documento word. Dopo circa 10 minuti vado ad aprire di nuovo la finestra del browser di HA e..."Connessione persa. Riconnessione in corso...".

**Reply by snakuzzo** [reply]
Prova a fare questo...
Su HA OS vai sotto Impostazion > Sistema > Rete
Espandi IPv6 e disabilitalo.
Clicca su Salva.

Vedi se ti ricapita

**Reply by 50494554524F** [reply]
è normalissimo, penso dipenda dallo sleep delle schede del browser ma non ho smanettato per sistemare... puoi provare a metterle nella white list

**Reply by Valerio0188** [reply]
> Prova a fare questo...
Su HA OS vai sotto Impostazion > Sistema > Rete
Espandi IPv6 e disabilitalo.
Clicca su Salva.


@snakuzzo non so da quale cilindro hai tirato fuori questa soluzione, ma sembra aver funzionato!
Continuerò a verificare se ricapita, ma intanto grazie mille!

**Reply by snakuzzo** [reply]
Bene ...buono a sapersi :)

**Comment by triak3ak** [comment]
Buongiorno a tutti, stesso identico problema. Ma il disabilitare l'IPv6 non ha risolto..
Tra l'altro noto che se non ho interagito da molto con la GUI di Home Assistant anche le mie richieste verso Google Assistant falliscono dicendomi che è impossibile contattare la mia istanza... (non c'è una regola.. è abbastanza random la cosa).
Poi apro la GUI di HA, vedo il messaggio "connessione persa, riconnessione in corso" e se poi faccio una richiesta di nuovo a Google Assistant funziona.
Uso Home Assistant OS su Raspberry Pi4 con IP statico impostato direttamente in HA.
Non riesco a capire quale sia il problema, grazie in anticipo a chi potrebbe darmi l'illuminazione risolutiva.

**Reply by Valerio0188** [reply]
Stai usando una VPN?

**Reply by triak3ak** [reply]
No, nessuna VPN. Ho ip pubblico fisso. Utilizzo DuckDNS giusto per avere un DNS di facile lettura e per utilizzare i certificati di Let's Encrypt. La mia è una configurazione molto base.

**Reply by crazyhorse1994** [reply]
Ciao, stesso identico problema, hai risolto? Grazie mille.

**Reply by Valerio0188** [reply]
> Ciao, stesso identico problema, hai risolto? Grazie mille.

Io uso una VPN ed il problema si presenta quando è attiva. Se la disattivo il problema non si presenta.

**Reply by triak3ak** [reply]
> Ciao, stesso identico problema, hai risolto? Grazie mille.

Ciao, no.. non ancora. Mi viene il dubbio che sia Eolo..
Però proverò a disabilitare uno ad uno i componenti aggiuntivi e vedere se magari c'è qualche cosa che influenza questo comportamento o meno..

**Reply by fabrymal** [reply]
ciao, io avevo lo stesso problema, a dir la verità non so come ho risolto, ma da quando ho eliminato i messaggi di errore nel log, ( ne avevo tanto e ripetitivi), e risolvendo i problemi che li generavano il mio HA non si disconnette più.
